src/olympia/reviews/serializers.py

```diff
@@ -34,6 +38,15 @@ def validate(self, data):
         # Also include the user ip adress.
         data['ip_address'] = request.META.get('REMOTE_ADDR', '')
 
+        # Clean up body and automatically flag the review if an URL was in it.
+        # Unquote the body in case someone tries 'example%2ecom'.
+        body = unquote(data.get('body', ''))
+        if '<br>' in data:
+            self.cleaned_data['body'] = re.sub('<br>', '\n', body)
```

Comment:
that's a very naive cleanup, django does have helpers that do that in a more reasonable fashion, maybe we should use those?